pkg/operator/ceph/object/user.go

```diff
@@ -185,6 +197,11 @@ func DeleteUser(c *Context, id string, opts ...string) (string, error) {
 		if code, ok := exec.ExitStatus(err); ok && code == int(syscall.ENOENT) {
 			return result, nil
 		}
+
+		res, er := ListUserBuckets(c, id)
```

Comment:
```suggestion
		res, err := ListUserBuckets(c, id)
```
Use the naming convention `err` for errors/

Reply:
Seb, I am doing it intentionally. If you see the below code block We have 2 errors and I want to keep them in separate variables and I have already used err in line no. 194. If you have something else in mind other than err let me know.

https://github.com/rook/rook/blob/53da581d754a61360f4d4b49281533c2d60461c5/pkg/operator/ceph/object/user.go#L194-L204

Reply:
Oops, I overlooked that :)

Reply:
@iamniting How about adding a comment about this? Or maybe call it `innerErr` so it's more obviously not a typo. :)

Reply:
changed to innerErr